src/main/java/com/greenfoxacademy/opal/kalendaryo/kalendaryo/model/entity/UserModel.java

```diff
@@ -62,16 +63,4 @@ public void setUserEmail(String userEmail) {
         this.userEmail = userEmail;
     }
 
-    public String getAccessToken() {
-        return accessToken;
-    }
-
-    public void setAccessToken(String accessToken) {
-        this.accessToken = accessToken;
-    }
-
-    public void setEmailAndToken(String userEmail, String accessToken) {
-        setUserEmail(userEmail);
-        setAccessToken(accessToken);
-    }
```

Comment:
Still move this to a service:  
         SecureRandom secureRandom = new SecureRandom();
         byte[] random = new byte[20];
         secureRandom.nextBytes(random);
         return Base64.encodeBase64String(random);